release-content/0.15/release-notes/_release-notes.toml

```diff
@@ -40,6 +40,13 @@ contributors = ["@kristoff3r", "@tychedelia", "@JMS55"]
 prs = [14876]
 file_name = "14876_Add_Order_Independent_Transparency.md"
 
+[[release_notes]]
```

Comment:
We should probably move this down (it's not the most important change), but we need to do a reordering pass anyways.

Reply:
Sure, that's up to you.

I just put it at the end of the graphics/rendering related stuff. The order: "Required components (most important change) -> a bunch of graphics/visual related stuff -> this PR (vaguely related to drawing/visuals) -> other stuff" made sense to me. :)

But feel free to reorder it wherever you like.